jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -42,18 +37,20 @@ Evaluations can cover many aspects of a model performance, including:
 * Respond in a fair and unbiased way, and within a specific style
 * Avoid certain expressions
 
-== Automating evaluations with CircleCI
+Using an open source library or third-party tools can simplify defining evaluations, tracking progress, and reviewing evaluation results.
 
-Evaluations can be expressed as classic software tests, typically characterised by the "input, expected output, assertion" format, and as such they can be automated into CircleCI pipelines.
+== Automating evaluations with CircleCI
 
-Two important differences between evals and classic software tests to keep in mind:
+*Evaluations can be automated in CircleCI pipelines.* Automating evaluations with CircleCI can enable you to automatically determine whether or not to accept proposed AI application changes, or deploy and release your AI application.
+There are important differences between evals and standard software tests to keep in mind:
 
-* LLMs are predominantly non-deterministic, leading to flaky evaluations, unlike deterministic software tests.
-* Evaluation results are subjective. Small regressions in a metric might not necessarily be a cause for concern, unlike failing tests in regular software testing.
+* LLMs are predominantly non-deterministic, leading to flaky evaluations, more so than typical deterministic software tests.
+* Evaluation results are subjective. Small regressions in a metric might not necessarily be a cause for concern, unlike failing tests in standard software testing.
+* Evaluation results are manually reviewed to determine whether AI application performance meets expectations since the output metric values can fluctuate.
```

Comment:
I think this section should go in the Evaluations overview section above as it's not about CircleCI specifics?

Reply:
lines 45-49